src/js/controller/captions.js

```diff
@@ -104,12 +103,13 @@ define(['utils/helpers',
         function _addTrack(track) {
 
             track.data = track.data || [];
+            track.name = track.label || track.name || track.language;
 
-            if (!track.label) {
-                track.label = 'Unknown CC';
+            if (!track.name) {
+                track.name = 'Unknown CC';
                 _unknownCount++;
                 if (_unknownCount > 1) {
-                    track.label += ' (' + _unknownCount + ')';
+                    track.name += ' (' + _unknownCount + ')';
```

Comment:
This and `_createLabel` in `tracks-mixins.js` are pretty similar. Anything we can do to consolidate them?